bedrock/settings/base.py

```diff
@@ -1302,5 +1302,6 @@ def lazy_email_backend():
     'activist': '014783244707853607354:bbpl3emdsii',
 }
 
-if config('SWITCH_FLASHTALKING', default=DEV, cast=bool):
-    CSP_IMG_SRC += ('servedby.flashtalking.com',)
+if config('SWITCH_TRACKING_PIXEL', default=DEV, cast=bool):
+    CSP_IMG_SRC += ('ad.doubleclick.net',
```

Comment:
@pmac - this isn't critical as we're going to remove the flash talking pixel pretty quickly, but it would be nice if there was a simple way to use the switch here too instead of add both domains to the white list. Is this possible?